image_to_gridmap:
  image_topic: "/image_publisher/image"
  resolution: 0.1
  map_frame_id: "map"
  min_height: -0.1
  max_height: 0.1
  
grid_map_visualization:
  grid_map_topic: /image_to_gridmap/grid_map
  grid_map_visualizations:
    - name: elevation_points
      type: point_cloud
      params:
        layer: elevation
    - name: elevation_grid
      type: occupancy_grid
      params:
        layer: elevation
        data_min: -0.2
        data_max: 0.2
    - name: flat_grid
      type: flat_point_cloud
      params:
        height: 0.0